Sources/ArcGISToolkit/Components/Authentication/NetworkChallengeContinuation.swift

```diff
@@ -71,6 +71,8 @@ extension NetworkChallengeContinuation.Kind {
             self = .login
         case .clientCertificate:
             self = .certificate
+        @unknown default:
+            fatalError("Unknown NetworkAuthenticationChallenge.Kind")
```

Comment:
I figure the message may be helpful if this is ever hit but I'd be fine with removing or changing it.